packages/core/src/element/node.ts

```diff
@@ -48,11 +49,19 @@ const singleNode: ShapeOptions = {
   },
   // 私有方法，不希望扩展的节点复写这个方法
   getLabelStyleByPosition(cfg: NodeConfig, labelCfg: ILabelConfig): LabelStyle {
+    const labelMaxLength = labelCfg.style?.maxLength;
+
+    let text = cfg!.label as string;
+
+    if (labelMaxLength) {
```

Comment:
Do we have any mechanism of seeing the full label or do we care about the full label at all?

Reply:
@Verdagio No, if you specify a max label length then there is no way of seeing the full label. I remember we debated whether a tooltip was required, but we agreed that could be added at a later stage if necessary.